sdk/anomalydetector/ai-anomaly-detector/review/ai-anomaly-detector.api.md

```diff
@@ -27,19 +27,30 @@ export interface AnomalyContributor {
 }
 
 // @public
-export class AnomalyDetectorClient {
-    constructor(endpointUrl: string, credential: TokenCredential | KeyCredential, options?: AnomalyDetectorClientOptions);
-    deleteMultivariateModel(modelId: string, options?: DeleteMultivariateModelOptions): Promise<void>;
-    detectAnomaly(modelId: string, detectionRequest: DetectionRequest, options?: DetectAnomalyOptions): Promise<AnomalyDetectorClientDetectAnomalyResponse>;
-    detectChangePoint(body: DetectChangePointRequest, options?: DetectChangePointOptions): Promise<AnomalyDetectorClientDetectChangePointResponse>;
-    detectEntireSeries(body: DetectRequest, options?: DetectEntireSeriesOptions): Promise<AnomalyDetectorClientDetectEntireResponse>;
-    detectLastPoint(body: DetectRequest, options?: DetectLastPointOptions): Promise<AnomalyDetectorClientDetectLastPointResponse>;
-    exportModel(modelId: string, options?: ExportModelOptions): Promise<AnomalyDetectorClientExportModelResponse>;
-    getDetectionResult(resultId: string, options?: GetDetectionResultOptions): Promise<AnomalyDetectorClientGetDetectionResultResponse>;
-    getMultivariateModel(modelId: string, options?: GetMultivariateModelOptions): Promise<AnomalyDetectorClientGetMultivariateModelResponse>;
-    listMultivariateModel(options?: ListMultivariateModelOptionalParams): Promise<AnomalyDetectorClientListMultivariateModelResponse>;
-    listMultivariateModelNext(nextLink: string, options?: ListMultivariateModelNextOptionalParams): Promise<AnomalyDetectorClientListMultivariateModelNextResponse>;
-    trainMultivariateModel(modelRequest: AnomalyDetectorClientModelInfo, options?: TrainMultivariateModelOptions): Promise<AnomalyDetectorClientTrainMultivariateModelResponse>;
+export class AnomalyDetectorClient extends AnomalyDetectorClientBase {
```

Comment:
@lmazuel - FYI this is the API surface. The remaining manual code is just a class extending the generated client to add support for Key Credential.

The operations are all in the generated client `AnomalyDetectorClientBase`, no change in the name of the operations or return types.